src/cargo/core/workspace.rs

```diff
@@ -68,7 +68,10 @@ enum MaybePackage {
 pub enum WorkspaceConfig {
     /// Indicates that `[workspace]` was present and the members were
     /// optionally specified as well.
-    Root { members: Option<Vec<String>> },
+    Root {
+        members: Option<Vec<String>>,
+        exclude: Vec<String>,
```

Comment:
Perhaps we should check that `exclude` is a relative path, to be conservative? Absolute paths probably does not make sense in this context.